### PR TITLE
Block 14 docs (deleting/restoring forms, etc.)

### PR DESCRIPTION
**Pull request by ktuite**
Making this a separate PR so it's easier to handle with all the other open backend PRs

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -34,17 +34,18 @@ Here major and breaking changes to the API are listed by version.
...
-* Forms can be restored
-* List of deleted forms available
-* More stuff
 
-**Changed**:
-* Deleting a form is now a reversible action
-* Deleted forms are purged after 30 days (all related data including submissions is destroyed)
-* More stuff
+* New `?groupPaths` and `?splitSelectMultiple` options on [CSV export paths](/reference/submissions/submissions/exporting-form-submissions-to-csv) which aim to replicate ODK Briefcase export behavior. One simplifies nested path names and the other breaks select multiple options out into multiple columns.
+* New `?deletedFields` option on [CSV export](/reference/submissions/submissions/exporting-form-submissions-to-csv) which exports all previously known and deleted fields and data on the form.
```
[the remark this answers is not in the document]
Comment (issa-tseng) [reply docs/api.md:42]: i'm confused, i think they're there? https://github.com/getodk/central-backend/pull/449/files#diff-9eddf4dc51bf6a0125ca7fb094468ad284112270385c41cebce4f8f0a29620abR2128-R2130

do you mean in the text? i think i looked and i usually don't exhaustively document all the flags..

[the remark this answers is not in the document]
Comment (matthew-white) [reply docs/api.md]: Ah I see them now, looks great!

File: docs/api.md [hunk 2]
```diff
@@ -1085,6 +1099,27 @@ This endpoint supports retrieving extended metadata; provide a header `X-Extende
 + Response 403 (application/json)
     + Attributes (Error 403)
 
+### List all deleted Forms [GET /v1/projects/{projectId}/forms?deleted=true]
+
+_(introduced: Version 1.4)_
+
+This endpoint returns a list of the current soft-deleted Forms that appear in the Trash section. In addition to the normal `Form` values, each Form will also included when it was deleted (`deletedAt`) and its numeric ID (`id`) that can be used to restore the Form.
```
Comment (matthew-white) [inline docs/api.md:1107]: "include" instead of "included" I think

File: docs/api.md [hunk 3]
```diff
@@ -1310,7 +1345,19 @@ We use `PATCH` rather than `PUT` to represent the update operation, so that you
 
 #### Deleting a Form [DELETE]
 
-Only `DELETE` a `Form` if you are sure you will never need it again. If your goal is to prevent it from showing up on survey clients like ODK Collect, consider setting its `state` to `closing` or `closed` instead (see [Modifying a Form](/reference/forms/individual-form/modifying-a-form) just above for more details).
+When a Form is deleted, it goes into Trash section, but it can now be restored from the Trash. After 30 days in the Trash, the Form and all of its resources and submissions will be automatically purged. If your goal is to prevent it from showing up on survey clients like ODK Collect, consider setting its `state` to `closing` or `closed` instead (see [Modifying a Form](/reference/forms/individual-form/modifying-a-form) just above for more details).
```
Comment (matthew-white) [inline docs/api.md:1349]: "the" before "Trash section"

File: docs/api.md [hunk 4]
```diff
@@ -3931,12 +3978,20 @@ These are in alphabetic order, with the exception that the `Extended` versions o
 + createdAt: `2018-01-19T23:58:03.395Z` (string, required) - ISO date format
 + updatedAt: `2018-03-21T12:45:02.312Z` (string, optional) - ISO date format
 
+## Deleted Form (Form)
++ deletedAt: `2018-03-21T12:45:02.312Z` (string, required) - ISO date format
++ id: `42` (number, required) - Numeric ID as it is represented in the database.
```
Comment (matthew-white) [inline docs/api.md:3995]: What do you think about describing `id` in functional terms? Something like:

> Numeric ID that distinguishes the Form from other Forms in the Project with the same `xmlFormId`. This ID can be used to restore the Form.

**Review by matthew-white — APPROVED** [review]
Just reviewed 3cf0990c283cdd7e385f720f369518cabd9ccd38 and made a few small comments. Other things to consider adding:

- Add `form.restore` and `form.purge` to the list of audit log actions.
- Maybe document that `Actor`s can have a `deletedAt` field?

File: docs/api.md (hunk 4 above)
Comment (ktuite) [reply docs/api.md:3995]: great suggestion!

**Pull request merged**